Detections/W3CIISLog/ProxyShellPwn2Own.yml

```diff
@@ -0,0 +1,42 @@
+id: 968358d6-6af8-49bb-aaa4-187b3067fb95
+name: Exchange ProxyShell Pwn2Own
```

Comment:
We are looking to standardize on naming convention a bit going forward.  We are not finalized on the naming convention yet, but it may be something close to this pattern.  Can you see about adjusting a bit, if it makes sense, to map to this on the name value please - {Activity|Operation} {Filter|Pattern} {Context|Entity} {-|:|[]} {Hunting|Detection}.  Just an example, so tweak as you see firt, but maybe use something like - "Exchange SSRF Autodiscover ProxyShell - Detection".

Reply:
Updated in line with suggested convention